### PR TITLE
WIP - Bootstrap 4.x update

### PR DESCRIPTION
**Pull request by uaktags**
This will only include the Bootstrap 4 (bootswatch updates).

Some themes will have to get removed due to bootswatch not updating them.
I am using uaktags/explorer's 06d8229 as the base (at time of writing, this is latest master), so a number of things need to be stripped and NOT included:
- i18 support (this will be a later enhancement)
- Any logic or new settings that I created like dynamic social lists, etc.

### CODE REVIEW

**Comment by uaktags** [comment]
Looks like there's some glitches that need to be fixed in the block, tx, and address view.

**Comment by minkcrypto** [comment]
Any updates with this PR?

**Comment by uaktags** [comment]
I think it looks as good as it's probably going to, unless there's other minor things like what I found with the movement page. Bascially have to make sure there's no ``t("translatethis")`` functions from my master branch.

**Comment by TheHolyRoger** [comment]
I will load this up to test out soon

File: views/markets/ccex.pug [hunk 1]
```diff
@@ -3,108 +3,107 @@ extends menu
 block market_view
   .row
     .col-md-12  
-      .panel.panel-default
-        .panel-heading
-          strong #{settings.locale.ccex} - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+      .card.card-default
+        .card-header
+          strong= t('market.market_title',{market : exchange.graviex,coin :  marketdata.coin, exchange : marketdata.exchange })
```
Comment (TheHolyRoger) [inline views/markets/ccex.pug:8]: Not sure what to do with these ones? How did they work with the t() function?

File: views/markets/cryptsy.pug [hunk 2]
```diff
@@ -3,104 +3,104 @@ extends menu
 block market_view
   .row
     .col-md-12  
-      .panel.panel-default
-        .panel-heading
-          strong #{settings.locale.cryptsy} - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+      .card.card-default
+        .card-header
+          strong= t('market.market_title',{market : exchange.graviex,coin :  marketdata.coin, exchange : marketdata.exchange })
```
Comment (TheHolyRoger) [inline views/markets/cryptsy.pug:8]: Not sure what to do with these ones? How did they work with the t() function?

File: views/markets/empoex.pug [hunk 3]
```diff
@@ -3,88 +3,88 @@ extends menu
 block market_view
   .row
     .col-md-12  
-      .panel.panel-default
-        .panel-heading
-          strong #{settings.locale.empoex} - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+      .card.card-default
+        .card-header
+          strong= t('market.market_title',{market : exchange.graviex,coin :  marketdata.coin, exchange : marketdata.exchange })
```
Comment (TheHolyRoger) [inline views/markets/empoex.pug:8]: Not sure what to do with these ones? How did they work with the t() function?

File: views/markets/poloniex.pug [hunk 4]
```diff
@@ -43,45 +43,48 @@ block market_view
...
     });        
   .row
-    .hidden-xs.col-md-12  
-      #chart-panel.panel.panel-default
-        .panel-heading
-          strong #{settings.locale.poloniex} - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+    .d-none.col-md-12  
+      #chart-card.card.card-default
+        .card-header
+          strong= t('market.market_title',{market : exchange.graviex,coin :  marketdata.coin, exchange : marketdata.exchange })
```
Comment (TheHolyRoger) [inline views/markets/poloniex.pug:49]: Not sure what to do with these ones? How did they work with the t() function?

File: views/markets/poloniex.pug [hunk 5]
```diff
@@ -43,45 +43,48 @@ block market_view
...
-        .panel-body
+            span.fa.fa-th-list.pull-right.view-summary.iquidus.market-toggle(data-toggle='tooltip', data-placement='bottom',  title= market.view_summary) 
+        .card-body
           div#chart(style="width:100%;height:300px;")
-      #summary-panel.panel.panel-default
-        .panel-heading
-          strong #{settings.locale.poloniex} - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+      #summary-card.card.card-default
+        .card-header
+          strong= t('market.market_title',{market : exchange.graviex,coin :  marketdata.coin, exchange : marketdata.exchange })
```
Comment (TheHolyRoger) [inline views/markets/poloniex.pug:56]: Not sure what to do with these ones? How did they work with the t() function?

File: views/markets/poloniex.pug [hunk 6]
```diff
@@ -43,45 +43,48 @@ block market_view
...
               else
                 td.success #{marketdata.data.summary.percentChange} %
-    .hidden-md.hidden-lg.col-xs-12
-      .panel.panel-default
-        .panel-heading
-          strong Poloniex - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+    .d-md-none.d-lg-none.col-xs-12
+      .card.card-default
+        .card-header
+          strong= t('market.market_title',{market : exchange.graviex,coin :  marketdata.coin, exchange : marketdata.exchange })
```
Comment (TheHolyRoger) [inline views/markets/poloniex.pug:85]: Not sure what to do with these ones? How did they work with the t() function?

File: views/markets/yobit.pug [hunk 7]
```diff
@@ -3,85 +3,87 @@ extends menu
 block market_view
   .row
     .col-md-12  
-      .panel.panel-default
-        .panel-heading
-          strong #{settings.locale.yobit} - #{marketdata.coin}/#{marketdata.exchange} - #{settings.locale.mkt_hours}
+      .card.card-default
+        .card-header
+          strong= t('market.market_title',{
```
Comment (TheHolyRoger) [inline views/markets/yobit.pug:8]: Not sure what to do with these ones? How did they work with the t() function?

**Comment by TheHolyRoger** [comment]
Ok can't see anything else but those last t() functions in a few market files. I'm now running this branch (1.7.4) on the roger explorer here, looks much better :) https://explorer.theholyroger.com/

**Comment by TheHolyRoger** [comment]
Just realised - we're not using all these flag icons are we? Was that part of another thing to add icons to the network view?

**Comment by uaktags** [comment]
Looks good to me to bring over.
t() function was (i'm pretty sure) part of my i18next attempt (t = translate) so it'd would try to convert between the tag and the locale file.

**Comment by TheHolyRoger** [comment]
Found and fixed some issues with jqplot

**Pull request merged**